front/app/modules/commercial/insights/services/insightsInputs.ts

```diff
@@ -42,6 +42,7 @@ export function insightsInputsStream(
     apiEndpoint: `${API_PATH}/${getInsightsInputsEndpoint(insightsViewId)}`,
     ...streamParams,
     skipSanitizationFor: ['category'],
+    cacheStream: false,
```

Comment:
Why is it necessary to disable the cache for inputs and input streams?

Reply:
Because, having the same IDs as ideas, they get cached as ideas which causes issues when you try to browse ideas. Which is weird since I now include the full idea but it avoids crashing the project page when clicking the link.

Reply:
Pff, I'm starting to think we should really prioritize this caching issue...
 But good enough for now :)